src/react/AppProvider.jsx

```diff
@@ -62,19 +62,19 @@ export default function AppProvider({ store, children }) {
   });
 
   return (
-    <ErrorBoundary>
-      <AppContext.Provider
-        value={{ authenticatedUser, config, locale }}
-      >
-        <IntlProvider locale={locale} messages={getMessages()}>
+    <IntlProvider locale={locale} messages={getMessages()}>
```

Comment:
In case of error `ErrorBoundary` renders component with translated messages which need access to `react-intl`'s context to load translations.

That's also the reason why test for `ErrorBoundary` needed to be updated

Reply:
Great catch!